dipy/nn/evac.py

```diff
@@ -297,7 +298,7 @@ def __predict(self, x_test):
     def predict(self, T1, affine,
                 voxsize=(1, 1, 1), batch_size=None,
                 return_affine=False, return_prob=False,
-                largest_area=True):
```

Comment:
You might have to use `deprecate_params` since this method is available the 2 last release at least.

Reply:
to discuss

Reply:
here it is : https://github.com/dipy/dipy/blob/90489e87b5496e050d03b1b53be2ca4892982ff0/dipy/reconst/forecast.py#L49